### PR TITLE
- dpipe: Fixed error checking for setpgrp(). 

### PR DESCRIPTION
**Pull request by jguillaumes**
The setpgrp() function, called with void (no parameters) returns a PID, or a -1 in case of error. The comparison to zero was wrong and always failed.

### CODE REVIEW

File: src/vde_router/vde_headers.h [hunk 1]
```diff
@@ -22,6 +22,15 @@
 #define PROTO_TCP 6
 #define PROTO_UDP 17
 
+#define XSTR(x) STR(x)
+#define STR(x) #x
+
+#ifdef VDE_DARWIN
```
Comment (danielinux) [inline src/vde_router/vde_headers.h]: I think this was here for debug purposes. Can we get rid of it?

Comment (jguillaumes) [reply src/vde_router/vde_headers.h]: Oh, sure, yes! I'm sorry I forgot to delete those lines :(
Do you want to do it yourself or do you prefer I do it and resubmit the pull request?

Comment (danielinux) [reply src/vde_router/vde_headers.h]: simply commit on top of the branch, the PR gets updated automatically.

**Comment by danielinux** [comment]
merged, thank you.

**Comment by jguillaumes** [comment]
You're welcome!

**Pull request merged**